node/notify/notfiy.go

```diff
@@ -2,13 +2,13 @@ package notify
 
 import (
 	"github.com/Qitmeer/qitmeer/core/message"
-	"github.com/Qitmeer/qitmeer/core/types"
+	"github.com/Qitmeer/qitmeer/services/mempool"
 )
 
 // Notify interface manage message announce & relay & notification between mempool, websocket, gbt long pull
 // and rpc server.
 type Notify interface {
-	AnnounceNewTransactions(newTxs []*types.Tx)
+	AnnounceNewTransactions(newTxs []*mempool.TxDesc)
```

Comment:
The types re-define inside `mempool` breaks the dependencies isolation rules of qitmeer. please define the type inside `types` if the type shared between components. so that the multiple part of system don't depends on `mempool` package directly.